src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,10 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User userService = new UserService().findByEmail(email);
```

Comment:
First of all, it should be `user` variable name. Also, please make `UserService` instance a class member, as it brings a lot of benenfits.

Reply:
I didn't quite understand the 2nd part, I hope I did as you asked